nupic/research/frameworks/dendrites/dendrite_segments.py

```diff
@@ -60,20 +60,20 @@ def __init__(self, num_units, num_segments, dim_context, sparsity, bias=None):
         self.sparsity = sparsity
 
         # TODO: Use named dimensions.
```

Comment:
You mentioned in another PR you didn't want to use named dimensions. If that is the case, then better to remove this TODO